### DISCUSSION TITLE
Does not connect to git hub with copilot

### DISCUSSION THREAD

**Discussion by passstor** [community/community#47606] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Does not connect to git hub with copilt

**Comment by neurolimitless** [comment]
Same

**Reply by neurolimitless** [reply]
Goland editor

**Comment by tuves** [comment]
Thank you for reporting this. The bug fix has now been deployed to production and affected users should be able to connect. 

Due to the recency of this incident, we are still investigating the contributing factors and will provide a more detailed update on cause and remediation in the February Availability Report, which we will publish the first Wednesday of March.

We sincerely apologize for any stress and/or frustration this has caused.
